app/views/site/keyboardMoveHelp.scala

```diff
@@ -27,6 +27,22 @@ object helpModal {
         )
       )
     )
+  def puzzle(implicit ctx: Context) =
+    frag(
+      h2(trans.keyboardShortcuts()),
+      table(
+        tbody(
+          header(trans.navigateMoveTree()),
+          row(frag(kbd("←"), or, kbd("→")), trans.keyMoveBackwardOrForward()),
+          row(frag(kbd("h"), or, kbd("l")), trans.keyMoveBackwardOrForward()),
+          row(frag(kbd("↑"), or, kbd("↓")), trans.keyGoToStartOrEnd()),
+          row(frag(kbd("k"), or, kbd("j")), trans.keyGoToStartOrEnd()),
+          row(kbd("f"), trans.flipBoard()),
+          row(kbd("z"), trans.preferences.zenMode()),
+          row(kbd("?"), trans.showHelpDialog())
```

Comment:
Looks like we can add a few more hotkeys. Namely N, L, X, and Space. We can borrow a header from the function below to separate the hotkeys as well.

Reply:
What should the keys N,L,X and Space functions be?

Reply:
So the file `keyboard.ts` will describe what each hotkey does. For instance, [N will go to the next puzzle](https://github.com/lichess-org/lila/blob/ade7bdff082c79242ca047135a9aafe376af761f/ui/puzzle/src/keyboard.ts#L31-L33).

Reply:
Okay thank you - I will do this now.

Reply:
No problem, and mind including a screenshot?

Reply:
Sure